tac-github-process.md

```diff
@@ -4,8 +4,8 @@ The process that is currently in use by the TAC is as follows:
 
 1. The chair checks the issues and pull requests (PRs) to decide which, if any,
    should be on the next meeting agenda.
-2. All TAC members (not just the voting members) should have sent their
-   github ids to the chair and the Acting Secretary (Stephano) so any
+2. All [TAC members](CODEOWNERS) (not just the voting members) should have sent their
```

Comment:
Probably better not to link to CODEOWNERS here since you have added the better file (tac-composition.md). Currently CODEOWNERS seems to have non-TAC members, but maybe that's in error.
If you want to connect to CODEOWNERS maybe link it to `reviewers` 2 lines below.

Reply:
My plan A was I'd list all TAC members (not just the voting reps) in the new file.  The thing is I couldn't find an "official" & up-to-date list of *all* TAC members anywhere, so CODEOWNERS seemed like a decent approximation.

Do you know whether such list exists somewhere?  If so, I could amend `tac-composition.md` and change the link in `tac-github-process.md` accordingly.

Reply:
I see. I don't what the original text about members referred to. I think the only formalism is the voting members. Other contributors can probably opt-in to tracking the repo with the `watch` feature.

Reply:
If we're going to define members and voting members then line 10 needs to be updated to read 'Once PRs are approved by 2 TAC voting members...'

Reply:
I think it's more pragmatic, though, to only maintain a list of voting members in this repo and let any other contributors, participants, etc. use the github watch feature.